Neos.Neos/Classes/Service/ContentElementEditableService.php

```diff
@@ -27,50 +23,12 @@
  */
 class ContentElementEditableService
 {
-
-    /**
-     * @Flow\Inject
-     * @var PrivilegeManagerInterface
-     */
-    protected $privilegeManager;
-
-    /**
-     * @Flow\Inject
-     * @var AuthorizationService
-     */
-    protected $nodeAuthorizationService;
-
-    /**
-     * @Flow\Inject
-     * @var FusionHtmlAugmenter
-     */
-    protected $htmlAugmenter;
-
     /**
      * Wrap the $content identified by $node with the needed markup for the backend.
-     *
-     * @param NodeInterface $node
-     * @param string $property
-     * @param string $content
-     * @return string
+     * This method is extended by the Neos.Ui package via an aspect to add the needed markup for inline editing.
      */
-    public function wrapContentProperty(NodeInterface $node, $property, $content)
+    public function wrapContentProperty(NodeInterface $node, string $property, string $content): string
```

Comment:
yes this is indeed aop via:
https://github.com/neos/neos-ui/blob/e53f8e20ee70ca832828f033f5a69c8223a2deab/Classes/Aspects/AugmentationAspect.php#L142-L150

but in the backend we use `$content = $joinPoint->getAdviceChain()->proceed($joinPoint);` instead of `$joinPoint->getMethodArgument('content')` 

this seems odd we basically use `htmlAugmenter->addAttributes` twice - so removing the code here is not 100% the same -> it might still work though.

Reply:
Thx done